rpm/containers-common.spec

```diff
@@ -0,0 +1,237 @@
+Source6: %{github_containers}/image/%{image_branch}/docs/containers-registries.d.5.md
+Source7: %{github_containers}/image/%{image_branch}/docs/containers-signature.5.md
+Source8: %{github_containers}/image/%{image_branch}/docs/containers-transports.5.md
+Source9: %{github_containers}/image/%{image_branch}/registries.conf
+Source10: %{github_containers}/shortnames/%{shortnames_branch}/shortnames.conf
+Source11: %{github_containers}/image/%{image_branch}/default.yaml
+Source12: %{github_containers}/image/%{image_branch}/default-policy.json
+Source13: %{github_containers}/storage/%{storage_branch}/docs/containers-storage.conf.5.md
+Source14: %{github_containers}/storage/%{storage_branch}/storage.conf
+# Fetch RPM-GPG-KEY-redhat-release from the authoritative source instead of storing
+# a copy in repo or dist-git. Depending on distribution-gpg-keys rpm is also
+# not an option because that package doesn't exist on CentOS Stream.
+Source15: https://access.redhat.com/security/data/fd431d51.txt
+
+%description
+This package contains common configuration files and documentation for container
+tools ecosystem, such as Podman, Buildah and Skopeo.
+
+It is required because the most of configuration files and docs come from projects
+which are vendored into Podman, Buildah, Skopeo, etc. but they are not packaged
+separately.
+
+%package extra
+Summary: Extra dependencies for Podman and Buildah
+Requires: %{name} = %{epoch}:%{version}-%{release}
+Requires: container-network-stack
+Requires: oci-runtime
+Conflicts: podman < 5:5.0.0~rc4-1
+Recommends: composefs
+Recommends: crun
+Requires: (crun if fedora-release-identity-server)
+Requires: netavark >= 1.10.3-1
+Suggests: slirp4netns
+Requires: passt
+Requires: iptables
+Requires: nftables
+Recommends: qemu-user-static
+Requires: (qemu-user-static-aarch64 if fedora-release-identity-server)
+Requires: (qemu-user-static-arm if fedora-release-identity-server)
+Requires: (qemu-user-static-x86 if fedora-release-identity-server)
+
+%description extra
+This subpackage will handle dependencies common to Podman and Buildah which are
+not required by Skopeo.
+
+%prep
+%autosetup -Sgit %{name}-%{version_no_tilde}
+
+# containers-auth.json.5.md
+cp %{SOURCE1} docs/.
+
+# containers-certs.d.5.md
+cp %{SOURCE2} docs/.
+
+# containers-policy.json.5.md
+cp %{SOURCE3} docs/.
+
+# containers-registries.conf.5.md
+cp %{SOURCE4} docs/.
+
+# containers-registries.conf.d.5.md
+cp %{SOURCE5} docs/.
+
+# containers-registries.d.5.md
+cp %{SOURCE6} docs/.
+
+# containers-signature.5.md
+cp %{SOURCE7} docs/.
+
+# containers-transports.5.md
+cp %{SOURCE8} docs/.
+
+# registries.conf
+cp %{SOURCE9} .
+
+# shortnames.conf
+cp %{SOURCE10} 000-shortnames.conf
+
+# default.yaml
+cp %{SOURCE11} .
+
+# default-policy.json
+cp %{SOURCE12} policy.json
+
+# containers-storage.conf.5.md
+cp %{SOURCE13} docs/.
+
+# storage.conf
+cp %{SOURCE14} .
+
+# RPM-GPG-KEY-redhat-release
+cp %{SOURCE15} RPM-GPG-KEY-redhat-release
```

Comment:
@lsm5 would you mind removing all the cp statements above to avoid duplicate copying in the similar ways as it's done in https://gitlab.com/redhat/centos-stream/rpms/containers-common/-/blob/stream-container-tools-latest-rhel-9.0.0-beta/containers-common.spec?ref_type=heads#L77 ? What I mean is to leave the %prep section empty and do everything via 'install' in %install section to avoid double copying?

Reply:
will do. Your suggestion looks much better, thanks.

Reply:
done